src/page/Page.tsx

```diff
@@ -26,7 +26,7 @@ export const NeoPage = ({
   isLoaded = true, // Whether the page is loaded and the cards can be displayed.
   onPageLayoutUpdate = () => {}, // action to take when the page layout is updated.
 }) => {
-  const getReportIndex = (pagenumber: number, index: number) => {
+  const getReportIndex = (pagenumber: number, index: number | string) => {
```

Comment:
index should be `id` now, with `string` as the only allowed type

Reply:
done